### ISSUE TITLE
Moving focus to an element with no ToolTip opens an ancestor's ToolTip

### ISSUE THREAD

**Issue by SamBent** [dotnet/wpf#4614]
* .NET Core Version: 3.0+
* Windows version: all
* Does the bug reproduce also in WPF for .NET Framework 4.8?: Yes
* Is this bug related specifically to tooling in Visual Studio (e.g. XAML Designer, Code editing, etc...)? no
* Security issues and bugs should be reported privately, learn more via our [responsible disclosure guidelines](https://github.com/dotnet/wpf/blob/master/README.md#reporting-security-issues-and-security-bugs).

 
 
 **Problem description:**
1. Run sample app from #4612
2. Press TAB repeatedly to move focus to the TextBox labeled "No tool tip".  (It's inside a GroupBox)
3. Wait 5 sec
 
 **Actual behavior:** 
 The GroupBox's ToolTip appears.

 **Expected behavior:**
 No ToolTip appears.

Moving keyboard focus to an element should open the element's ToolTip if it has one.  It should not open some other element's ToolTip.

**Comment by SamBent** [comment]
Fixed by the tooltip work in 6.0-preview7.
